### PR TITLE
BME68x SPI, heater work, etc.

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.Enums.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
+﻿namespace Meadow.Foundation.Sensors.Atmospheric
+{
+    partial class Bme68x
+    {
+        /// <summary>
+		/// Valid addresses for the sensor.
+		/// </summary>
+		public enum Addresses : byte
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.Enums.cs:8]: Please fix indenting in these lines

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.HeaterProfileConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            /// The heater resistance.
+            /// </summary>
+            public ushort HeaterResistance { get; set; }
+            /// <summary>
+            /// The heater duration in the internally used format
+            /// </summary>
+            public TimeSpan HeaterDuration { get; set; }
+
+            /// <summary>
+            ///
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.HeaterProfileConfiguration.cs:26]: Add Summary text here, and maybe remove exception section comment?

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.Registers.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
+﻿namespace Meadow.Foundation.Sensors.Atmospheric
+{
+    partial class Bme68x
+	{
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.Registers.cs:4]: Fix indentation here.

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                base(spiBus, chipSelectPort)
+            { }
+        }
+    }
+    /// </summary>
+    /// <remarks>
+    /// This class implements the functionality necessary to read the temperature, pressure, humidity and VOS
+    /// from the Bosch BME680 sensor
+    /// </remarks>
+    public partial class Bme680 : Bme68x
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs:56]: Move this to a separate file.

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+namespace Meadow.Foundation.Sensors.Atmospheric
+{
+    /// <summary>
+    /// BME680 Temperature, Pressure, Humidity and gas sensor
+    /// </summary>
+    /// <remarks>
+    /// This class implements the functionality necessary to read the temperature, pressure, humidity and VOS
+    /// </remarks>
+    namespace Meadow.Foundation.Sensors.Atmospheric
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs:11]: Why encapsulate this in another namespace (which is the same name)?

**Review by jorgedevs — CHANGES_REQUESTED** [review]

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs (hunk 5 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs:11]: yeah that was copy/paste mistake - I'm shocked it compiles - I'll fix :)

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.Enums.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Bme68x.Enums.cs]: fixed!

File: Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs (hunk 4 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Bme68x/Driver/Drivers/Bme680.cs:11]: a mistake - deleted

**Review by jorgedevs — APPROVED** [review]

**Pull request merged**